chain/chain/src/chain.rs

```diff
@@ -2573,14 +2573,12 @@ impl Chain {
         let sync_prev_block = self.get_block(sync_block_header.prev_hash())?;
```

Comment:
I'll skip for now because as Marcelo showed that this code part shouldn't be changed, so I reverted it. The fix is only related to receipts aggregation